solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -345,8 +345,6 @@ contract RandomBeacon is IRandomBeacon, IApplication, Governable, Reimbursable {
         address indexed operator
     );
 
-    event RewardsWithdrawn(address indexed stakingProvider, uint96 amount);
```

Comment:
Just to confirm, do we really want to remove this event? Why we don't need it anymore?

Reply:
It is no longer needed because it is [now declared in `IApplication` interface](https://github.com/threshold-network/solidity-contracts/blob/47268908a1aa92ab215b76e5f26730519643b663/contracts/staking/IApplication.sol#L25-L26) and is a part of [`withdrawRewards` function contract](https://github.com/threshold-network/solidity-contracts/blob/47268908a1aa92ab215b76e5f26730519643b663/contracts/staking/IApplication.sol#L31-L32). After updating `@threshold-network/solidity-contracts` dependency, the compilation failed because `RandomBeacon` extends `IApplication` and that event is already defined in `IApplication`. See https://github.com/threshold-network/solidity-contracts/pull/95 please.